code/__DEFINES/mobs.dm

```diff
@@ -313,6 +313,11 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 #define IS_MOTH (1<<7)
 //=================================================
 
+//AFK status
+#define MOB_CONNECTED 0
+#define MOB_AFK 1 //Still within the grace period.
```

Comment:
AFK isn't AFK here, its disconnected but within grace.

I would personally find another name as its confusing. To me AFK is still connected, but not moved for X time.
However from your implementation it looks like its disconnected within 5/15 mins.

Reply:
Which name would you give it?

Reply:
Changed it to `MOB_RECENTLY_DISCONNECTED`

Reply:
Sorry i didn't have a name at the time, `DISCONNECTING` would also work